src/Console/Report/FixReport/ReporterFactory.php

```diff
@@ -25,19 +24,18 @@
 final class ReporterFactory
 {
     /**
-     * @var ReporterInterface[]
+     * @var array<string, ReporterInterface>
      */
     private array $reporters = [];
 
     public function registerBuiltInReporters(): self
     {
-        /** @var null|string[] $builtInReporters */
+        /** @var null|list<string> $builtInReporters */
```

Comment:
why?

Reply:
You have already started that discussion in #6293, please see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6293#discussion_r820157214 and let me know whether that convinces you or do you still want me to revert these changes :)